src/sql/workbench/contrib/query/browser/queryActions.ts

```diff
@@ -873,6 +883,11 @@ export class ListDatabasesActionItem extends Disposable implements IActionViewIt
 		this._dropdown.value = databaseName;
 		this._dropdown.values = [databaseName];
 		this._dropdown.enabled = true;
+
+		// Set the strict selection to false so that it is allowed to not to have a database selected to begin with.
```

Comment:
can the comment also mention that this was added to support server level MySQL queries for more context when people are looking at this code in the future?